### ISSUE TITLE
Text nearly unreadable: "vouchers and waiting list reservations"

### ISSUE THREAD

**Issue by rixx** [pretix/pretix#1051]
In the quota overview, a chart is presented showing where different parts of the quota are used. As of recently, one part of that chart is called "vouchers and waiting list reservations", which doesn't really fit inside the circle.
![screenshot_2018-10-20 quota](https://user-images.githubusercontent.com/2657547/47246045-13ab1a80-d3fc-11e8-83d9-60d501a2ca4f.png)

**Comment by raphaelm** [comment]
Yeah. It's not trivial to fix this but still have the full text in the table, but it should be easy enough.
